templates/inspec/examples/google_pubsub_subscription.erb

```diff
@@ -0,0 +1,5 @@
+<% gcp_project_id = grab_attributes['gcp_project_id'] -%>
+<% subscription = grab_attributes['subscription'] -%>
+describe google_pubsub_subscription(project: <%= doc_generation ? "'#{gcp_project_id}'" : "gcp_project_id" -%>, name: <%= doc_generation ? "'#{subscription['name']}'" : "subscription['name']" -%>) do
```

Comment:
I've recently changed how attributes, tests and docs are compiled to *hopefully* make it a little less redundant and verbose. I'm happy to refactor this after the merge, just a FYI